packages/amplication-git-pull-remote/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@amplication/git-pull-remote",
```

Comment:
please rename it to git pull service, I do want to have a services naming convention for service names suffix of "-service"